x-pack/legacy/plugins/epm/public/screens/detail/screenshots.tsx

```diff
@@ -58,13 +58,15 @@ export function Screenshots(props: ScreenshotProps) {
           {/* By default EuiImage sets width to 100% and Figure to 22.5rem for size=l images,
               set image to same width.  Will need to update if size changes.
             */}
-          <EuiImage
-            url={toImage(image.src)}
-            alt="screenshot image preview"
-            size="l"
-            allowFullScreen
-            style={{ width: '22.5rem', maxWidth: '100%' }}
-          />
+          {image.src && (
+            <EuiImage
```

Comment:
The [EPR type](https://github.com/elastic/package-registry/pull/196/files#diff-d96ec7ae5bb4a0907779f25ed03acb33R15) and [golang code](https://github.com/elastic/package-registry/blob/master/util/package.go#L63) have `src` as optional